src/Dialogs/AbstractPropertiesDialog.vala

```diff
@@ -140,7 +140,10 @@ protected abstract class Files.View.AbstractPropertiesDialog : Granite.Dialog {
             storage_levelbar.get_style_context ().add_class ("inverted");
 
             var storage_label = new Gtk.Label (
-                _("%s free out of %s").printf (format_size (fs_capacity - fs_used + fs_reserved), format_size (fs_capacity))
+                _("%s free out of %s").printf (
+                    format_size (fs_capacity - fs_used + fs_reserved),
+                    format_size (fs_capacity)
+                )
```

Comment:
You shouldn't make changes which are unrelated to the issue in hand - however tempting it may be!  You can always submit a separate "clean up the code" PR which makes no substantive changes but improves the formatting.  Its best to keep the size of diff as small as possible.